wasm/read.go

```diff
@@ -25,3 +26,21 @@ func readString(r io.Reader, n int) (string, error) {
 	}
 	return string(bytes), nil
 }
+
+func readU32(r io.Reader) (uint32, error) {
+	var buf [4]byte
+	_, err := io.ReadFull(r, buf[:])
+	if err != nil {
+		return 0, err
+	}
+	return binary.LittleEndian.Uint32(buf[:]), nil
+}
+
+func readU64(r io.Reader) (uint64, error) {
+	var buf [4]byte
```

Comment:
b/c there isn't any *yet*. at least not for `wasm` itself.
(I was in the process of adding some. see go-interpreter/wagon#19)